### DISCUSSION TITLE
発表スライドイメージ

### DISCUSSION THREAD

**Discussion by NoRio-kd** [TeamC-by-TNI/Felizo#77] [Q&A] [answered]
叩き台です！テキスト（タイトル、メッセージ、本文）はまだ推敲できてません！
今日の発表内容とmergeする形で下記スライドを混ぜれれば。
没にすることや修正になんの抵抗もないので忌憚のない意見をください。
#プロダクトについての説明
<img width="782" alt="image" src="https://github.com/user-attachments/assets/88113d68-9f5f-4c90-b12e-3dbded04944a" />
<img width="787" alt="image" src="https://github.com/user-attachments/assets/ce047b78-4a03-4f1a-a5a8-68b968b1bdd2" />
<img width="780" alt="image" src="https://github.com/user-attachments/assets/f43e0071-7907-4b84-a70e-8c25eda98deb" />
<img width="781" alt="image" src="https://github.com/user-attachments/assets/51e4f205-e29f-41c1-beeb-f5898493aeff" />

下記の要素はまだですがよしなに記載します。上記で何がご意見あれば！
#Why me
#苦労した点
#良かった点

**Comment by mikiIDE** [comment]
仕事が早い…！ありがとうございます！
ちょっと明日のデバッグ次第で実装の内容が変わるかと思うので😭
その時に相談させてほしい！

**Comment by tkg-geek** [comment, marked as answer]
ありがとうございます。
Goodeデザインですね！

発表内容で「役割分担」もありますが、これは今日のスライドを適当に流用してもらえたらと思います。
<img width="815" alt="スクリーンショット 2024-12-24 17 29 35" src="https://github.com/user-attachments/assets/e33946fc-61a4-4907-a2ca-5a610a6a2502" />

また、#良かった点（主に進め方）として、僕は以下があるなと思っています。参考までに。
・メンバー全員がゼロベースなので、初期から一般的な作業の仕方を確認しながら進められた。おそらく一般的な開発と齟齬なく、変わった独自ルールを設けずやれている。
・GitHub Organizationの学習も兼ねてできる限り活用した。（どうやら我々が一番使ってる気がします）
・スタート時の環境最適化が良かった。作業手順の明文化（ReadME）、リポジトリ作成と同時にdevをさくらサーバーに自動デプロイ化（GitHub ActionsでCI/CD）、議事録やメモの残し方共有、など。（このあたり行ったり来たりせずに進めた）
・概ねGit Flow形式で進められた（※意外と他チームは理解していない）　→参考(https://yubase-inc.notion.site/GitHub-8bd419383cd84dcbadd32846261b1911)

#苦労した点
・分担をどうするのが良いか、やってみないとわからなかった
・Breeze無しでの環境構築による対応（授業でBreeze前提でしかやってない）
・（その他お二人からもあると思うので相談しましょう）
